gst/nnstreamer/tensor_transform/tensor_transform.c

```diff
@@ -798,6 +804,24 @@ gst_tensor_transform_set_property (GObject * object, guint prop_id,
       filter->acceleration = FALSE;
 #endif
       break;
+    case PROP_APPLY:
+    {
+      gint64 val;
+      const gchar *param = g_value_get_string (value);
+      gchar **strv = g_strsplit_set (param, ",", -1);
+      guint i, num = g_strv_length (strv);
+
+      for (i = 0; i < num; i++) {
+        val = g_ascii_strtoll (strv[i], NULL, 10);
```

Comment:
You should set `errno == 0` before calling those functions setting `errno` internally.

Reply:
Also, `ERANGE` only handles overflows. You should check other violations. Please check the glib document.

Reply:
Updated.

Reply:
Well.. you didn't provide `endptr` as an argument. So, the third check is meaningless.

Reply:
Right.. I missed it. Updated.